### ISSUE TITLE
Incompatible with Ruby >=3.2.0

### ISSUE THREAD

**Issue by Meantub** [gosuri/vagrant-env#16]
Ruby deprecated the `File.exists?` that the old `dotenv` version this package was using and it causes the plugin to not work. This repo is pretty stale so I fixed it in my fork if people need it, I'm not a Ruby developer so it might be wrong but it worked for me

https://github.com/Meantub/vagrant-env

**Comment by zekefarwell** [comment]
I just upgraded Vagrant and am also running into this issue.  Any chance of a PR from @Meantub's fork getting merged?

**Comment by FrancoisClementKW** [comment]
> Ruby deprecated the `File.exists?` that the old `dotenv` version this package was using and it causes the plugin to not work. This repo is pretty stale so I fixed it in my fork if people need it, I'm not a Ruby developer so it might be wrong but it worked for me
> 
> https://github.com/Meantub/vagrant-env

Thanks for your work.

When I run `vagrant plugin install vagrant-env`, vagrant always install the source one…
How can I install your plugin instead of the source one?

**Comment by Meantub** [comment]
@FrancoisClementKW You have to build my branch, then `vagrant plugin install <.gem file>`

**Comment by wolkenarchitekt** [comment]
> @FrancoisClementKW You have to build my branch, then `vagrant plugin install <.gem file>`

What are the exact steps to build the forked plugin, for someone who has never manually build a vagrant plugin?

**Comment by FrancoisClementKW** [comment]
> @FrancoisClementKW You have to build my branch, then `vagrant plugin install <.gem file>`

Thanks, but I'm not familiar with Ruby, and I don't have any tool to build a `.gem` file.
I eventually decided to use a yaml configuration file instead of a `.env` file.

**Comment by atoolchain** [comment]
hey @FrancoisClementKW, @wolkenarchitekt & who else needs a little help
i have been able to install it, with some playing around with the https://github.com/Meantub/vagrant-env repo.
side note: i am not a ruby dude...

git clone https://github.com/Meantub/vagrant-env.git
cd ./vagrant-env
gem build vagrant-env.gemspec
vagrant plugin install ./vagrant-env-0.0.4.gem


maybe you have to uninstall the old 0.0.3 plugin first (i did)
worked for me :)

**Comment by Meantub** [comment]
Yeah sorry I didn't respond, I also am not a ruby dev and only did it to fix my own problem a year ago so didn't remember either
